nodeshift/resource/gpu/gpu.go

```diff
@@ -55,10 +57,21 @@ func (r *gpuResource) Schema(c context.Context, request resource.SchemaRequest,
 			KeyGPUCount: schema.Int64Attribute{
 				Description: DescriptionGPUCount,
 				Optional:    true,
+				Computed:    true,
+				Default:     int64default.StaticInt64(1),
 			},
 			KeyRegion: schema.StringAttribute{
 				Description: DescriptionRegion,
 				Optional:    true,
+				Computed:    true,
+			},
+			KeyDiskSizeGB: schema.Int64Attribute{
+				Description: DescriptionDiskSizeGB,
+				Optional:    true,
```

Comment:
Better to make it required for terraform users

Reply:
done